src/ui/components/Video/RecordedCursor.tsx

```diff
@@ -18,8 +22,8 @@ export function RecordedCursor() {
       ({ currentTime, graphicsRect, localScale, recordingScale }) => {
         const { height, width } = graphicsRect;
 
-        const mouseEvent = findMostRecentMouseEvent(currentTime);
-        const clickEvent = findMostRecentClickEvent(currentTime);
+        const mouseEvent = findMostRecentMouseEvent(currentTime, focusWindow?.begin.time);
+        const clickEvent = findMostRecentClickEvent(currentTime, focusWindow?.begin.time);
```

Comment:
I think this change should be reverted because of its impact on all non-Cypress recordings.

Thinking about other ways we might convey the information of "the cursor hasn't moved in a while" that wouldn't _break_ the interaction for non-Cypress cases: Perhaps we could add a threshold (similar to `CLICK_TIMING_THRESHOLD_MS`) of a couple of seconds after which we _dimmed_ the cursor icon? (Not sure if that would be intuitive enough though.)

Reply:
You've got the better product sense here, but I thought about this while I was working on it and "hide the cursor any time it happened before the focus window" feels like a _potentially_ valid thing to do.

Reply:
Yeah it's plausible for sure. It's not crazy. (Especially in the context of Cypress.) I just think it's the _more_ confusing of the two directions we could go.

Thanks for being flexible 🙇🏼